nvi

```diff
@@ -91,6 +91,7 @@ function install_node() {
   NODE_VERSION=$1
   DOWNLOAD_DIR=$2
   INSTALL_DIR=$3
+  local PACKAGE="node-v${NODE_VERSION}-linux-x64"
```

Comment:
Indeed.

Mainstream distros are moving towards abandoning 32-bit but that doesn't affect existing images.